kivy/tests/test_graphics_svg.py

```diff
@@ -0,0 +1,92 @@
+'''
+Svg tests
+==============
+
+Testing Svg rendering.
+'''
+
+import sys
+import pytest
+from kivy.tests.common import GraphicUnitTest, requires_graphics
```

Comment:
`sys`, `pytest` and `requires_graphics` are not needed here.

Reply:
Fixed.